wire-ios/Wire-iOS/Resources/Base.lproj/Localizable.strings

```diff
@@ -454,6 +454,16 @@
 // Both placesholders contains a localized list of names, e.g "Alice, Bob, and Cate"
 "content.system.missing_messages.users_added_and_removed.plural" = "You haven’t used this device for a while. Some messages may not appear here.\n\nMeanwhile, %1$@ have been added. %2$@ have been removed.";
 
+// MLS Migration
+"content.system.mls_migration.started.description" = "The standard messaging protocol is changing from Proteus to the new Messaging Layer Security (MLS).";
```

Comment:
Will the other languages be imported automatically at some point?

Reply:
IIRC it is done by this action https://github.com/wireapp/wire-ios/actions/workflows/crowdin.yml

Reply:
I was told that is it done automatically via GitHub actions